src/bin/nydus-image/main.rs

```diff
@@ -1164,6 +1190,26 @@ impl Command {
         Ok(())
     }
 
+    fn chunkdict_generate(matches: &ArgMatches) -> Result<()> {
+        // Connecting database and Generating chunk dictionary by algorithm "exponential_smoothing"
+        let db_url: &String = matches.get_one::<String>("database").unwrap();
+        debug!("db_url: {}", db_url);
+        let algorithm = String::from("image_and_verion_deduplicate");
```

Comment:
Can users specify the algorithm name from the CLI option, and why not call it `exponential_smoothing`?